### PR TITLE
[#22] Tests for Login/Logout UI

### PR DESCRIPTION
**Pull request by ironfroggy**
Adds login/logout tests for standard users (#22)

Assumes CLOUDIGRADE_BASE_URL env var is set properly.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -58,23 +55,37 @@ jobs:
     - stage: test-cloudigrade
       env:
         - NAME=ui-chrome
+      addons:
```
Comment (elyezer) [inline .travis.yml]: Much better! Thank you

File: integrade/tests/ui/test_login.py [hunk 2]
```diff
@@ -13,16 +13,15 @@
...
+from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 
 from widgetastic.browser import Browser
 
 from integrade.config import get_config
 from integrade.utils import base_url
 
-from .utils import find_element_by_text, \
+from .utils import find_element_by_text, wait_for_input_value, \
```
Comment (elyezer) [inline integrade/tests/ui/test_login.py]: Usually we use parenthesis when import items does not fit on a line. But using `\` is OK too.

File: Makefile [hunk 3]
```diff
@@ -32,6 +32,12 @@ test:
 test-api:
 	py.test $(PYTEST_OPTIONS) integrade/tests/api/v1
 
+test-ui:
+	py.test $(PYTEST_OPTIONS) integrade/tests/ui --driver=Firefox
```
Comment (elyezer) [inline Makefile]: Any particular reason to prefer Firefox? (I am not saying that we should use any particular browser, I'm just curious why you chosen Firefox :smile:)

Comment (ironfroggy) [reply Makefile]: I just hadn't installed Chrome yet, actually. I ran it on Chrome later once all the tests were in place, but Firefox > Chrome for my personal taste so I already had it!

**Pull request merged**